app/services/backorder_updater.rb

```diff
@@ -28,7 +28,7 @@ def amend_backorder(order)
 
     backorder = orderer.find_open_order(order)
 
-    update(backorder, user, distributor, order_cycle)
+    update(backorder, user, distributor, order_cycle) if backorder
```

Comment:
I guess it amounts to the same thing, but I thought that we usually avoid implied booleans and use the more explicit `present?`

But i'm sure it amounts to the same thing, and this looks cleaner so I like it :)